.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+<!-- Don't forget a short, descriptive title! -->
+
+# What's in this PR?
+
+<!-- List the changes you made and your reasons for them.-->
+
+<!-- Make sure any changes to code include changes to documentatino.-->
```

Comment:
Okay, so my thinking here is that while editing they will see this and then if they don't delete the comments after adding their details, it won't show up in the issue/PR. I've seen that happen a lot on templates when someone forgets. So that's why it doest appear in the preview.